### PR TITLE
Feature/page example

### PR DESCRIPTION
**Pull request by Shipow**

### CODE REVIEW

**Comment by redox** [comment]
:+1: @Shipow

File: docs/_layouts/default.html [hunk 1]
```diff
@@ -42,7 +42,7 @@
 {% endif %}
 
 {% if site.env == 'development' %}
-    <script src="//cdn.jsdelivr.net/scrollmagic/2.0.5/plugins/debug.addIndicators.min.js"></script>
+    <!-- <script src="//cdn.jsdelivr.net/scrollmagic/2.0.5/plugins/debug.addIndicators.min.js"></script> -->
```
Comment (vvo) [inline docs/_layouts/default.html]: Can we remove this if not used/not working? (I had errors on it at some point)

**Comment by Shipow** [comment]
If you can keep it as comment to allow esay debug it's fine, if it you still have error, yes you can remove it.

**Comment by vvo** [comment]
Ah ok, WFM

**Pull request merged**